apps/user-service/src/auth/dto/reset-password-response.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { JsonApiAttributes } from "@terramatch-microservices/common/dto/json-api-attributes";
+
+@JsonApiDto({ type: 'logins', id: 'number' })
+export class ResetPasswordResponseDto extends JsonApiAttributes<ResetPasswordResponseDto> {
+  @ApiProperty({
+    description: 'User id',
```

Comment:
Let's be careful with language around "id"s. This is a UUID, not an ID (IDs are numbers)